### ISSUE TITLE
Huge memory usage

### ISSUE THREAD

**Issue by akirmse** [mikereedell/sunrisesunsetlib-java#15]
I'm seeing production crashes where the Location class is trying to allocate 320kb, presumably for either the latitude or the longitude.  Is there a reason why unlimited precision is necessary?  Wouldn't double be good enough?

**Comment by MenoData** [comment]
I agree that `double` would be good enough. In general: The accuracy of astronomical calculations cannot be increased by choosing `BigDecimal` but by choosing another better astronomical algorithm.
